### DISCUSSION TITLE
Quick question, is there a way...

### DISCUSSION THREAD

**Discussion by myndxero** [adieyal/sd-dynamic-prompts#365] [Q&A] [answered]
... to comment things out in the text file so dynamic prompts ignores it? I tried to search, but admittedly I give up fast on those, so apologies if it's' been asked or in an obvious place.

I ask because I like to keep my text file lists complete though sometimes some of the items contained I don't want showing up in oone run or another.

Thank you so much!

**Comment by akx** [comment, marked as answer]
Yes – for text files, empty lines and lines that start with `#` are ignored.

You can also use YAML, which may be more comfortable – see https://github.com/adieyal/sd-dynamic-prompts/blob/main/collections/publicprompts.yaml for an example.

**Reply by myndxero** [reply]
> with `#` are ignored.

thank you so much, tis is exactly what I needed.
